Description: Fix builds of qtconcurrentthreadengine.h with GCC 11
 Without this patch, all the code that #includes QtConcurrent headers
 will fail to compile with GCC 11.
Origin: upstream, https://codereview.qt-project.org/c/qt/qtbase/+/339417
Last-Update: 2022-04-01

--- a/src/concurrent/qtconcurrentthreadengine.h
+++ b/src/concurrent/qtconcurrentthreadengine.h
@@ -247,8 +247,8 @@ template <>
 class ThreadEngineStarter<void> : public ThreadEngineStarterBase<void>
 {
 public:
-    ThreadEngineStarter<void>(ThreadEngine<void> *_threadEngine)
-    :ThreadEngineStarterBase<void>(_threadEngine) {}
+    ThreadEngineStarter(ThreadEngine<void> *_threadEngine)
+        : ThreadEngineStarterBase<void>(_threadEngine) {}
 
     void startBlocking()
     {
